pkg/services/sqlstore/migrations/annotation_mig.go

```diff
@@ -109,4 +110,51 @@ func addAnnotationMig(mg *Migrator) {
 	//
 	updateEpochSql := "UPDATE annotation SET epoch = (epoch*1000) where epoch < 9999999999"
 	mg.AddMigration("Convert existing annotations from seconds to milliseconds", NewRawSqlMigration(updateEpochSql))
+
+	//
+	// 6.3: Make Regions a single annotation row
+	//
+	mg.AddMigration("Add epoch_end column", NewAddColumnMigration(table, &Column{
+		Name: "epoch_end", Type: DB_BigInt, Nullable: false, Default: "0",
```

Comment:
Think this needs to be nullable true to handle existing data is this table. Integration tests probably works because there are no data in the table, but I need to verify this by manual testing.

Reply:
I tested with SQLite and it works... but yes, we need to manually test other databases migrations.  May be good to have our tests start with something and try to migrate... but beyond my skills now :)